docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
       interval: 1m
       timeout: 50s
       retries: 3
-    image: aeternity/ae_mdw:latest
+    image: aeternity/ae_mdw:master
```

Comment:
This should stay “latest”

Reply:
The latest tag doesn't exist on dockerhub, and it needs to be built locally every time. And if users want to run `aeternity/ae_mdw` they won't be able to. We should probably figure out a way to build on a per-version basis (not branch) and create latest tag as well. I'll revert it back to `latest`